controllers/BookmarkController.php

```diff
@@ -38,6 +38,14 @@ public function actionUpdate($id)
             throw new HttpException(404, Yii::t('ContentModule.base', 'Invalid content id given!'));
         }
 
-        $content->bookmark();
+        if ($content->bookmark()) {
+            if ($content->isBookmarked()) {
+                $this->getView()->success(Yii::t('ContentModule.base', 'Content successfully bookmarked. You can access it directly from your profile.'));
+            } else {
+                $this->getView()->info(Yii::t('ContentModule.base', 'Content successfully removed from bookmarks.'));
+            }
+        } else {
+            $this->getView()->error(Yii::t('ContentModule.base', 'You cannot bookmark this content!'));
```

Comment:
Please use Yii::t Category  ContentBookmarkModule.base

Reply:
done https://github.com/humhub/content-bookmarks/pull/7/commits/95d1dc8fb2e2e9ce8c8f13bd59d3a2835577fa59

Reply:
fixed to `ContentBookmarksModule.base` in commit https://github.com/humhub/content-bookmarks/pull/7/commits/a56aedfc1bd85cc4288235c6166bd58b2d8404e7 because tests were broken here https://github.com/humhub/content-bookmarks/runs/2202198718?check_suite_focus=true